presto-native-execution/presto_cpp/main/PrestoExchangeSource.h

```diff
@@ -58,6 +58,11 @@ class PrestoExchangeSource : public velox::exec::ExchangeSource {
   /// PrestoExchangeSource.
   static void getMemoryUsage(int64_t& currentBytes, int64_t& peakBytes);
 
+  /// Invoked to reset the node-wise peak memory usage back to the current
+  /// memory usage in PrestoExchangeSource. This is used to track the peak
+  /// memory usage in recent time period instead of all time peak.
+  static void resetPeakMemoryUsage();
```

Comment:
```
/// Invoked to reset the node-wise peak memory usage back to the current
/// memory usage in PrestoExchangeSource. Instead of getting all time peak, 
/// this can be useful when tracking the peak within some fixed time intervals.
```